src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -1025,6 +1040,23 @@ export default class LazyMobXTable<T> extends React.Component<
                     ) : (
                         ''
                     )}
+                    {showResetColumnsButton && (
+                        <div>
+                            <button
+                                className="btn btn-default btn-sm pull-right"
+                                onClick={() => {
+                                    this.store.resetColumnVisibility();
+                                    if (this.props.storeColumnVisibility) {
+                                        this.props.storeColumnVisibility(
+                                            this.store.columnVisibility
+                                        );
+                                    }
+                                }}
+                            >
+                                Reset Columns
+                            </button>
+                        </div>
```

Comment:
Should we move this button inside the column selector as a third button (named `Select Default` or similar) next to `Select all` and `Deselect all` buttons?

Also, resetting the columns re-renders the entire mutation mapper, probably because we re-initiate the mutation mapper store.

Reply:
One reason I'm hesitant to do that is because that "Select default" button should probably only affect those columns in the given tab, which is potentially counter-intuitive and also more difficult to implement. The original ticket was to design a button which resets the column visibility of the table, so a user would have to flick through all the tabs (of which there are only currently 2) and click that "Select default" button in order to achieve this.

Also, I think the re-rendering may be necessary in cases where the initial column visibility didn't include lazy-loaded columns like "Annotation", so that the data in those columns is properly loaded and sorted once "Reset Columns" is clicked.